woocommerce/woocommerce-gutenberg-products-block
# Revert "Move cart and mini-cart blocks into their own folders (#6126)"

This reverts commit 069cf33a775a042f7bf0f26986704b627c875acf. 

There was a merge conflict with some of @ralucaStan's work on the cart order summary inner blocks that caused some of the inner blocks to go missing. This fixes it and will address the moving of the `shared` and `mini-cart` folders in a different PR

### Manual Testing

How to test the changes in this Pull Request:

1. Load the cart block page
2. The Coupon, tax and subtotal inner blocks should be present
